lib/ui/widgets/record_attendance.dart

```diff
@@ -21,14 +21,72 @@ class RecordAttendance extends StatelessWidget {
     }
   }
 
+  final reasonTextFieldController = TextEditingController();
+  static const String REASON_CHECKIN_LATE = 'Reason checkin late';
```

Comment:
CHECKIN_LATE_REASON or LATE_REASON_LABEL

'Reason checkin late' -> 'Reason for being late'